<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_combined" stretch="1,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1,1,1,0">
        <item>
         <widget class="QRadioButton" name="radioButton_bidT">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>입찰공고 물품</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_bidC">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>입찰공고 공사</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_bidS">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>입찰공고 용역</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_bidF">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>입찰공고 외자</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_bidE">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>입찰공고 기타</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_login">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1,1">
        <item>
         <widget class="QRadioButton" name="radioButton_ctrT">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>계약현황 물품</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_ctrC">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>계약현황 공사</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_ctrS">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>계약현황 용역</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_ctrF">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>계약현황 외자</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="9" column="0" rowspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,2,0,0,2,4,2,0,3,3,2,0">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>수요기관</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_dminsttNm"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>검색범위</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit_start">
        <property name="date">
         <date>
          <year>2020</year>
          <month>6</month>
          <day>23</day>
         </date>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>~</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit_end">
        <property name="date">
         <date>
          <year>2020</year>
          <month>6</month>
          <day>23</day>
         </date>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_minVal">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="inputMask">
         <string>00000</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="maxLength">
         <number>5</number>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="cursorPosition">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>억원 이상만</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>0페이지(0건)중 페이지</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_curPage">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_move">
        <property name="text">
         <string>이동</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,0,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item row="4" column="5">
       <widget class="QPushButton" name="pushButton_del">
        <property name="text">
         <string>DB reset</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>DB다운로드</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QComboBox" name="comboBox">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAlphabetically</enum>
        </property>
        <item>
         <property name="text">
          <string>지역 선택</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>전국</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>서울특별시</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>부산광역시</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>대구광역시</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>인천광역시</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>광주광역시</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>대전광역시</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>울산광역시</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>세종특별자치시</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>경기도</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>강원도</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>충청북도</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>충청남도</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>전라북도</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>전라남도</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>경상북도</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>경상남도</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>제주도</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>기타</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>입찰명</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="12" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="9" column="0">
       <widget class="QPushButton" name="pushButton_3months">
        <property name="text">
         <string>최근3개월</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,0,1">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>검색시간</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_time"/>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pushButton_today">
        <property name="text">
         <string>오늘</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>검색범위</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Root User</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="pushButton_1month">
        <property name="text">
         <string>최근1달</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="pushButton_3days">
        <property name="text">
         <string>최근 3일</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="pushButton_1week">
        <property name="text">
         <string>최근1주일</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="10">
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_login</sender>
   <signal>released()</signal>
   <receiver>pushButton_login</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1006</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>977</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
